srcpkgs/tasksh/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'tasksh'
+pkgname=tasksh
+version=1.2.0
+revision=1
+build_style=cmake
+makedepends="libreadline8"
```

Comment:
This looks wrong, you need -devel packages to be able to link libraries.

Reply:
Yeah, it was definitely wrong. I changed it to `readline-devel`, however I can't figure out it the executable was built with readline. The documentation for building is scarce. But steps in the building process and generated dependencies seem to indicate that it is the case.

Nonetheless, the AUR package doesn't include readline as a dependecy.